### PR TITLE
IDEX-3876: Change the titles and messages of all Che notifications

### PR DESCRIPTION
**Pull request by vzhukovskii**
@vparfonov @ashumilova

### CODE REVIEW

File: plugin-java/che-plugin-java-ext-debugger-java-client/src/main/resources/org/eclipse/che/ide/ext/java/jdi/client/JavaRuntimeLocalizationConstant.properties [hunk 1]
```diff
@@ -46,7 +46,7 @@ evaluateExpression = Evaluate Expression
 ############### Messages ################
 debugger.connecting = Connecting to the target VM...
 debugger.disconnecting = Disconnecting from the target VM...
-debugger.connected = Connected to the target VM, address: {0}
+debugger.connected = Connected to the target VM, address: {0}.
```
Comment (sleshchenko) [inline plugin-java/che-plugin-java-ext-debugger-java-client/src/main/resources/org/eclipse/che/ide/ext/java/jdi/client/JavaRuntimeLocalizationConstant.properties]: I think you should remove dot at the end of line. See https://docs.google.com/spreadsheets/d/1mNmRVN0jTWBjemb9U7JOvLLCXlvu2Jfa9EwPY2cnguc/edit#gid=0

File: plugin-java/che-plugin-java-ext-debugger-java-client/src/main/resources/org/eclipse/che/ide/ext/java/jdi/client/JavaRuntimeLocalizationConstant.properties [hunk 2]
```diff
@@ -46,7 +46,7 @@ evaluateExpression = Evaluate Expression
 ############### Messages ################
 debugger.connecting = Connecting to the target VM...
```
Comment (sleshchenko) [inline plugin-java/che-plugin-java-ext-debugger-java-client/src/main/resources/org/eclipse/che/ide/ext/java/jdi/client/JavaRuntimeLocalizationConstant.properties]: Also you should remove ... I think

File: plugin-java/che-plugin-java-ext-debugger-java-client/src/main/resources/org/eclipse/che/ide/ext/java/jdi/client/JavaRuntimeLocalizationConstant.properties (hunk 1 above)
Comment (vzhukovskii) [inline plugin-java/che-plugin-java-ext-debugger-java-client/src/main/resources/org/eclipse/che/ide/ext/java/jdi/client/JavaRuntimeLocalizationConstant.properties]: No, I shoudln't

Comment (sleshchenko) [inline plugin-java/che-plugin-java-ext-debugger-java-client/src/main/resources/org/eclipse/che/ide/ext/java/jdi/client/JavaRuntimeLocalizationConstant.properties]: OK, I understood. Please rename constant to something like "debugger.connected.description" and fix message in google doc

**Comment by ashumilova** [comment]
+1

**Pull request merged**